docs/src/modules/components/AppNavDrawer.js

```diff
@@ -202,7 +202,7 @@ function ProductDrawerButton(props) {
             role="menuitem"
             icon={<IconImage name="product-core" />}
             name="MUI Core"
-            description="Ready-to-use foundational components, free forever."
+            description="Foundational components for shipping features faster."
```

Comment:
A bit +1 to have "free forever" in the final description, it feels critical to mention.
Also "ready-to-use" speaks louder to me than "faster". It's clearer and still implicitly means faster. So personally, I would not do this change.

Reply:
In that case I think we should at least make a tiny punctuation adjustment:

> "Ready-to-use foundational components—free forever."

@oliviertassinari the original suggested change here is the same as the copy that we just shipped for the homepage. Do you think we should update that too? I feel like all of these one-line descriptions should be the same to better reinforce the brand identity.

Reply:
Should the one-line description also be used as the hero title? In some cases, using exactly the same would not work perfectly... The "—" gets weird. Or is it just me?!

<img width="350" alt="Screen Shot 2022-04-11 at 12 59 41" src="https://user-images.githubusercontent.com/67129314/162782110-11050be1-7748-4dd4-9ffe-78c86c26b500.png">

Reply:
I think the hero description should probably get its own custom-tailored copy to suit the page. I'm not sure that "free forever" needs to be included in the header. It's probably enough to just say it in the text below.

Reply:
Agreed, yeah 👍